uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/entitlement-api_test.js

```diff
@@ -0,0 +1,88 @@
+define([
+  'mocks/data/entitlement-data',
+  'leave-absences/shared/apis/entitlement-api',
+], function (mockData) {
+  'use strict';
+
+  describe('EntitlementAPI', function () {
+    var EntitlementAPI, $httpBackend;
+
+    beforeEach(module('leave-absences.apis'));
+
+    beforeEach(inject(function (_EntitlementAPI_, _$httpBackend_) {
+      EntitlementAPI = _EntitlementAPI_;
+      $httpBackend = _$httpBackend_;
+
+      //when the URL has this pattern
+      //civicrm/ajax/rest?action=get&entity=LeavePeriodEntitlement
+      //GET /civicrm/ajax/rest?action=get&entity=LeavePeriodEntitlement&json={"sequential":1,"api.LeavePeriodEntitlement.getremainder":{"entitlement_id":"$value.id","include_future":true}}&sequential=1
+      $httpBackend.whenGET(/action\=get&entity\=LeavePeriodEntitlement/)
+        .respond(function (method, url, data, headers, params) {
```

Comment:
the paths clash for both get and get with chained getremainder call (it was never getting called) so added filter in the same call